fiftyone/core/config.py

```diff
@@ -57,13 +57,22 @@ def __init__(self, d):
             env_var="FIFTYONE_DEFAULT_VIDEO_EXT",
             default=".mp4",
         )
+        self.default_window = self.parse_string(
+            d,
+            "default_window",
+            env_var="FIFTYONE_DEFAULT_WINDOW",
+            default="browser",
+        )
         self._show_progress_bars = None  # declare
         self.show_progress_bars = self.parse_bool(
             d,
             "show_progress_bars",
             env_var="FIFTYONE_SHOW_PROGRESS_BARS",
             default=True,
         )
+        self.do_not_track = self.parse_bool(
```

Comment:
Suggestion: add an FAQ question that references the existence of this parameter and describes how we track (anonymous) usage